backend/app/adapter/sqldb/migration/13_create_user_changelog_table.sql

```diff
@@ -2,7 +2,8 @@
 CREATE TABLE "user_changelog"
 (
     "user_id"              CHARACTER VARYING(5),
-    "email"                CHARACTER VARYING(254) NOT NULL PRIMARY KEY,
+    -- TODO: Add Primary Key Constraint to email on migrate up
```

Comment:
This should be in the standard format of `-- TODO(issue#000)` please make the issue and update this after.